### PR TITLE
[IMP] mail: add test for temporary or transient messages not have reply icon

### PR DESCRIPTION
**Pull request by bit-odoo**
Task-2702690

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/components/discuss/tests/discuss_tests.js [hunk 1]
```diff
@@ -1284,6 +1284,35 @@ QUnit.test('basic rendering of message', async function (assert) {
...
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/");
+    });
+    await afterNextRender(() =>
+        document.querySelector('.o_ComposerSuggestion').click()
+    );
+    await afterNextRender(() =>
+        document.querySelector('.o_Composer_buttonSend').click()
+    );
```
Comment (alexkuhn) [inline addons/mail/static/src/components/discuss/tests/discuss_tests.js:1307]: Put a comment above that these user interactions is to forge a transient message response from channel command "/who".

File: addons/mail/static/src/components/discuss/tests/discuss_tests.js [hunk 2]
```diff
@@ -1284,6 +1284,35 @@ QUnit.test('basic rendering of message', async function (assert) {
...
+    await afterNextRender(() => document.querySelector('.o_Message').click());
```
Comment (alexkuhn) [inline addons/mail/static/src/components/discuss/tests/discuss_tests.js:1308]: Put comment above to tell click on message to show actions on the transient message resulting from the "/who" command.

File: addons/mail/static/src/components/chat_window_manager/tests/chat_window_manager_tests.js [hunk 3]
```diff
@@ -1993,6 +1993,33 @@ QUnit.test('chat window with a thread: keep scroll position in message list on t
...
+        document.querySelector('.o_ComposerTextInput_textarea').dispatchEvent(kevt);
+    });
+    await afterNextRender(() => document.querySelector('.o_Message').click());
+    assert.containsNone(
+        document.body,
+        '.o_MessageActionList_actionReply',
+        "should not have action to reply to temporary/transient messages"
+    );
+});
+
```
Comment (alexkuhn) [inline addons/mail/static/src/components/chat_window_manager/tests/chat_window_manager_tests.js:2022]: I would just drop this test with chat window and just keep the test in discuss app.
It's duplicate with discuss with slight feature differences that are quite unrelated to the test at hand.

**Review by alexkuhn — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/discuss/tests/discuss_tests.js [hunk 4]
```diff
@@ -1284,6 +1284,34 @@ QUnit.test('basic rendering of message', async function (assert) {
...
+    //These user interactions is to forge a transient message response from channel command "/who"
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/who");
+    });
+    //Click on message to show actions on the transient message resulting from the "/who" command
+    await afterNextRender(() =>
+        document.querySelector('.o_Composer_buttonSend').click()
+    );
+    await afterNextRender(() => document.querySelector('.o_Message').click());
```
Comment (alexkuhn) [inline addons/mail/static/src/components/discuss/tests/discuss_tests.js:1307]: The comment should be above the click on `.o_Message`, not `.o_Composer_buttonSend` (which is a user interaction related to forging the transient message response).

File: addons/mail/static/src/components/discuss/tests/discuss_tests.js [hunk 5]
```diff
@@ -1284,6 +1284,34 @@ QUnit.test('basic rendering of message', async function (assert) {
...
+                default_active_id: 'mail.channel_20',
+            },
+        },
+    });
+    //These user interactions is to forge a transient message response from channel command "/who"
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/who");
+    });
+    //Click on message to show actions on the transient message resulting from the "/who" command
```
Comment (alexkuhn) [inline addons/mail/static/src/components/discuss/tests/discuss_tests.js:1303]: Missing whitespace immediately after `//` for comments.

**Comment by bit-odoo** [comment]
@alexkuhn changes are done

**Review by alexkuhn — APPROVED** [review]
@robodoo r+
Thanks!
